Cargo.toml

```diff
@@ -22,11 +22,12 @@ serde = { version = "~1.0.91", features = ["derive"] }
 serde_json = "~1.0.39"
 structopt = "~0.2.15"
 rcgen = "~0.7.0"
-rustls = "~0.16.0"
+rustls = { version = "~0.16.0", features = ["dangerous_configuration"] }
```

Comment:
That seem unfortunate in a production configuration.

Reply:
I don't think the "dangerous_configuration" means "don't use this in production". It rather means "you should know what you are doing".